src/renderer/dx/DxFontRenderData.cpp

```diff
@@ -729,16 +865,10 @@ Microsoft::WRL::ComPtr<IDWriteTextFormat> DxFontRenderData::_BuildTextFormat(con
 
     // If the OS supports IDWriteTextFormat3, set the font axes
     ::Microsoft::WRL::ComPtr<IDWriteTextFormat3> format3;
-    if (!FAILED(format->QueryInterface(IID_PPV_ARGS(&format3))) && !_axesMap.empty())
+    if (!FAILED(format->QueryInterface(IID_PPV_ARGS(&format3))) && !_axesVector.empty())
     {
-        std::vector<DWRITE_FONT_AXIS_VALUE> axesVector;
-        for (const auto [axis, value] : _axesMap)
-        {
-            const auto dwriteTag = DWRITE_MAKE_FONT_AXIS_TAG(gsl::at(axis, 0), gsl::at(axis, 1), gsl::at(axis, 2), gsl::at(axis, 3));
-            axesVector.push_back(DWRITE_FONT_AXIS_VALUE{ dwriteTag, gsl::narrow<float>(value) });
-        }
-        DWRITE_FONT_AXIS_VALUE const* axesList = &gsl::at(axesVector, 0);
-        format3->SetFontAxisValues(axesList, gsl::narrow<uint32_t>(axesVector.size()));
+        DWRITE_FONT_AXIS_VALUE const* axesList = &gsl::at(_axesVector, 0);

```

Comment:
This is weird. Why not just use `_axesVector.data()`?

Reply:
The analyzer complains :/ I suppose we could do what we did in `CustomTextLayout` and suppress the warning

Reply:
Yeah it just feels like overkill to go through the `gsl::at` when we know there's data and the size of it.